.vscode/launch.json

```diff
@@ -12,6 +12,14 @@
       "args": ["run", "-c", "local.yml"],
       "cwd": "${workspaceFolder}",
       "preLaunchTask": "cargo build"
+    },
+    {
+      "type": "lldb",
+      "request": "attach",
+      "name": "Attach to running process",
+      "program": "${workspaceFolder}/target/debug/symbolicator",
+      "stopOnEntry": true,
+      "pid": "${command:pickMyProcess}"
```

Comment:
Do we need both `program` and `pid`?

Reply:
otherwise it will throw an error if there are multiple instances running, like they are for symbolicator…